### ISSUE TITLE
pluginNames.map is not a function

### ISSUE THREAD

**Issue by GioUrnau** [emberjs/ember.js#17817]
Sou novo com o ember. Quando tentei instalar deu esse erro. O que pode ser?

**Comment by kiwiupover** [comment]
Im new to ember. When I tried to install gave this error. What can it be? English translation from Portuguese

Thanks for reporting @GioUrnau

**Comment by pzuraq** [comment]
Hey @GioUrnau! Sorry about this issue, it looks like this may have been a bug in `ember-cli-babel@v7.7.2`. Can you try updating your dependencies? It should be fixed in `v7.7.3`.

in `package.json`:

```json5
{
  // ...
  "dependencies": {
    // ...
    "ember-cli-babel": "^7.7.3"
  }
}
```

**Comment by rwjblue** [comment]
I think this is all good to go (happy to reopen if I'm wrong)...
